website/docs/introduction.md

```diff
@@ -10,42 +10,64 @@ This is a **very early and incomplete alpha release** of Docusaurus 2. We are ma
+
+While our main focus will still be helping you get your documentations right and well, the shared nature of building this static site generator makes it possible for you to build your whole website interface one-stop.
 
 ## Features
 
-- ⚛️ **Built using React** - Extend or customize with React
-- 📦 **Uses webpack** - Build an optimized bundle and use any CSS-in-JS solution you like
-- ⚡️ **Lightning Fast** - Link prefetching, instant navigation and page views
-- ✂️ **Easy** - Hot reloading out-of-the-box, automatic route-based code and data splitting! Create new routes by creating new Markdown or JSX files
-- 💥 **Single-page Application** - Client-side navigations and easily add dynamic interactivity to your website (Client-side Rendering)
-- 🎯 **SEO Friendly** - `index.html` files are statically generated for every possible path (Static Rendering)
-- 😌 **Painless** - Painless project setup. First class support for documentation
-- 📝 **MDX Based** - Write interactive components via JSX and React embedded in markdown
-- ☁️ **GitHub pages friendly** - Publish to GitHub pages with one command!
-- 🔌 **Pluggable** - Plugin system to extend basic functionality and do almost anything youw ant
-- 🎨 **Themeable** - Theming system to customize how your website's appearance
-- 🔍 **Search** - Make it easy for people to search what they need in your website.
-- 🌎 **i18n** - Internationalize your website easily (_coming soon_)
-- 💾 **Versioning** - Versioning support (_coming soon_)
-- 🚀 **Many others** ....
+Docusaurus is built with high attention to your experience building your site and maintaining it with your collaborators and contributors.
+
+- ⚛️ **Built with 💚 and React**
+  - Extend and customize with React
+  - Gain full control of your site's browsing experience by `swizzling` in your own components
+- 🔌 **Pluggable**
+  - Bootstrap your site with a basic template, then pick and plug functionalities built by us and our community.
+  - Open source your plugins to share with your fellow documentarians, because sharing is caring.
+- ✂️ **Developer experience**
+  - Multiple bootstrapping templates to get your site up and running, start writing your docs right now
+  - Universal configuration entry point to make it easily maintainable by contributors
+  - Hot reloading with lightning fast incremental build on changes
+  - Route-based code and data splitting
+  - Publish to GitHub Pages, Netlify, and other deployment services with ease
+
+Our shared goal — to help your users find what they need fast, and understand your products better. With the experience of Docusaurus 1, we share with you our best practices to help you build your doc site right and well.
+
+- 🎯 **SEO friendly**
+  - `index.html` files are statically generated for every possible path
```

Comment:
rather than index.html files, html files is the correct term.

path with ending slash / is converted to index.html
path without ending slash is .html